lib/compat/wordpress-6.9/rest-api.php

```diff
@@ -29,3 +29,22 @@ function gutenberg_rest_theme_export_link_rel( $response, $theme ) {
 	return $response;
 }
 add_filter( 'rest_prepare_theme', 'gutenberg_rest_theme_export_link_rel', 10, 2 );
+
+/**
+ * Overrides the REST controller for the attachment post type to add support
+ * for filtering by multiple media types.
+ *
+ * Only applies if the experimental media processing feature is not enabled,
+ * as that feature includes this functionality and more.
+ *
+ * @param array  $args      Array of arguments for registering a post type.
+ * @param string $post_type Post type key.
+ * @return array Modified array of arguments.
+ */
+function gutenberg_override_attachments_rest_controller( $args, $post_type ) {
+	if ( 'attachment' === $post_type && ! gutenberg_is_experiment_enabled( 'gutenberg-media-processing' ) ) {
```

Comment:
In order to avoid collisions with the media processing experiment which also overrides the same class, we check whether it's enabled before overriding. 

I also suffixed the class with `_6_9` to avoid redeclaring `Gutenberg_REST_Attachments_Controller` which already exists in the media processing experiment.